src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -436,6 +456,56 @@ impl DeviceManager {
 
         Err(anyhow!("ID are exhausted"))
     }
+
+    async fn try_update_device(&mut self, update_config: &DeviceConfig) -> Result<()> {
```

Comment:
Can you rename `update_config` to `updated_config` here to make it clearer?

Reply:
Yeah updated_config seems better. I'll fix it.
Thx

Reply:
> Can you rename update_config to updated_config here to make it clearer?

@amshinde Not sure to see how this is clearer : the contract of this function is clearly to update the device with a possibility of failure. Imperative tense looks appropriate, and this is what all other `try_.*_device()` functions do.

Reply:
@gkurz I was referring to the parameter `update_config` and not the function name. Of course imperative tense makes sense for functions but not the function parameters.